browser/brave_wallet/solana_provider_renderer_browsertest.cc

```diff
@@ -1096,26 +1096,110 @@ IN_PROC_BROWSER_TEST_F(SolanaProviderRendererTest, NonConfigurable) {
 IN_PROC_BROWSER_TEST_F(SolanaProviderRendererTest, Block3PIframe) {
```

Comment:
maybe we should rename this to just `3PIframe` or similar.

Reply:
rebased to `Iframe3P`